src/elm/Pages/MyAccount/View.elm

```diff
@@ -13,19 +12,27 @@ import User.Model exposing (..)
 view : WebData User -> Html a
 view user =
     let
-        name =
+        ( name, login, avatar ) =
             case user of
-                Success user' ->
-                    user'.name
+                Success val ->
+                    let
+                        name' =
+                            case val.name of
+                                Just name ->
+                                    name
+
+                                Nothing ->
+                                    val.login
+                    in
+                        ( name', val.login, img [ src val.avatarUrl ] [] )
 
                 _ ->
-                    ""
+                    ( "", "", div [] [] )
     in
-        div [ class "ui icon message" ]
-            [ i [ class "user icon" ]
-                []
+        div [ class "ui centered card" ]
+            [ div [ class "image" ] [ avatar ]
             , div [ class "content" ]
-                [ text <| "Welcome " ++ name
-                , p [] [ text "This is an account page" ]
+                [ div [ class "header" ] [ text <| "Welcome " ++ name ]
+                , div [ class "meta" ] [ text <| login ]
```

Comment:
maybe change to `[text <| "@" ++ login]`

(btw, if you didn't have this change it would be enough to do `[ text login ]`